shared/message/input/GameState.proto

```diff
@@ -55,33 +62,54 @@ message GameState {
             MANUAL                        = 10;
             PLAYER_PUSHING                = 11;
         }
-
+        /// A tracking message for robots, with penalty tracking data
         message Robot {
-            uint32                    id             = 1;
-            PenaltyReason             penalty_reason = 2;
-            google.protobuf.Timestamp unpenalised    = 3;
-        }
 
-        message Team {
-            uint32         team_id       = 1;  // unique team number
-            uint32         score         = 2;  // team's score
-            string         coach_message = 3;  // the coach's message to the team
-            repeated Robot players       = 4;
+            /// The robot which is being tracked
+            uint32 id = 1;
+            /// The infraction the robot being tracked committed
+            PenaltyReason penalty_reason = 2;
+            /// The timestamp when the robot will be playing again
+            google.protobuf.Timestamp unpenalised = 3;
         }
+        /// Set of robots on a team, with a score and coach message
+        message Team {
 
```

Comment:
This newline looks a bit strange to me, it's in a few other places in the proto documentation too.